samples/sensor/paj7620_gesture_polling/sample.yaml

```diff
@@ -0,0 +1,10 @@
+sample:
+  name: PAJ7620 gesture polling sample
+tests:
+  sample.sensor.paj7620_gesture_polling:
+    build_only: true
+    tags: sensors
+    platform_allow: nucleo_f334r8
+    depends_on:
+      - i2c
+      - paj7620
```

Comment:
This tag doesn't exist. You should use a filter instead. Like this: https://github.com/zephyrproject-rtos/zephyr/blob/846b4687154af5f57ba5a32610ec868d5f4ae61d/samples/sensor/max6675/sample.yaml#L14

Same for the other sample